test/HostnameTest.php

```diff
@@ -506,6 +506,25 @@ public function testAdditionalUTF8TLDs()
         }
     }
 
+    public function testAdditionalPunycodedTLDs()
+    {
+        $validator = new Hostname(Hostname::ALLOW_ALL);
+        // Check UTF-8 TLD matching
+        $valuesExpected = [
```

Comment:
When refactor please add a description for each case using the key of the array

`['invalid foo' => 'xn--3-owe4au9mpa.xn--xkc2al3hye2a']`